pkg/reconciler/kubernetes/tektonconfig/extension.go

```diff
@@ -45,9 +47,48 @@ func (oe kubernetesExtension) PreReconcile(context.Context, v1alpha1.TektonCompo
 }
 func (oe kubernetesExtension) PostReconcile(ctx context.Context, comp v1alpha1.TektonComponent) error {
 	configInstance := comp.(*v1alpha1.TektonConfig)
-	if configInstance.Spec.Profile == common.ProfileAll {
+
+	switch configInstance.Spec.Profile {
+	case common.ProfileAll:
 		return extension.CreateDashboardCR(comp, oe.operatorClientSet.OperatorV1alpha1())
+
+	case common.ProfileBasic:
+		_, err := extension.GetDashboard(oe.operatorClientSet.OperatorV1alpha1().TektonDashboards(), common.DashboardResourceName)
+		if err != nil {
+			if errors.IsNotFound(err) {
+				return nil
+			}
+			return err
+		}
+		return extension.TektonDashboardCRDelete(oe.operatorClientSet.OperatorV1alpha1().TektonDashboards(), common.DashboardResourceName)
+
+	case common.ProfileLite:
+		_, err := extension.GetDashboard(oe.operatorClientSet.OperatorV1alpha1().TektonDashboards(), common.DashboardResourceName)
+		if err != nil {
+			_, err := tt.GetTrigger(oe.operatorClientSet.OperatorV1alpha1().TektonTriggers(), common.TriggerResourceName)
```

Comment:
triggers is common to both platform could the deletion logic be in reconciler instead of extension 🤔

Reply:
yes I think it makes sense to add check for triggers in reconciler 👍🏻